tests/api2/test_305_container.py

```diff
@@ -136,36 +143,422 @@ def test_10_get_new_public_image_id(request):
+                    'name': 'Eric_Var',
+                    'value': 'Something'
+                }
+            ]
+        }
+    }
+    results = PUT(f'/chart/release/id/{tc_release_id}/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_35_verify_ix_chart_config_container_Command_Args_EnvironmentVariables(request):
+    depends(request, ['tc_containerCAE'])
+    results = GET(f'/chart/release/id/{tc_release_id}/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    config = results.json()['config']
+    assert 'sleep' in config['containerCommand'], results.text
+    assert 'infinity' in config['containerArgs'], results.text
+    assert {'name': 'Eric_Var', 'value': 'Something'} in config['containerEnvironmentVariables'], results.text
+
+
+@pytest.mark.dependency(name='hostpath-mountpath')
+def test_36_create_datasets_hostpath_and_mountpath(request):
+    depends(request, ['pool_04'], scope='session')
+    result = POST('/pool/dataset/', {'name': f'{pool_name}/tc-hostpath'})
+    assert result.status_code == 200, result.text
+    result = POST('/pool/dataset/', {'name': f'{pool_name}/tc-mountpath'})
+    assert result.status_code == 200, result.text
+
+
+@pytest.mark.dependency(name='tc_volumes')
+def test_37_set_truecommand_ix_chart_volumes(request):
+    depends(request, ['tc_chart_release'])
+    global payload
+    payload = {
+        'values': {
+            'volume': [
+                {
+                    'mountPath': f'/mnt/{pool_name}/tc-mountpath',
+                    'datasetName': f'{pool_name}/tc-mountpath',
+                }
+            ]
+        }
+    }
+    results = PUT(f'/chart/release/id/{tc_release_id}/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_38_verify_ix_chart_config_volumes(request):
+    depends(request, ['tc_volumes'])
+    results = GET(f'/chart/release/id/{tc_release_id}/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    payload_volume = payload['values']['volume'][0]
+    assert payload_volume in results.json()['config']['volume'], results.text
+
+
+@pytest.mark.dependency(name='tc_hostPathVolumes')
+def test_39_set_truecommand_ix_chart_hostPathVolumes(request):
+    depends(request, ['tc_chart_release'])
+    global payload
+    payload = {
+        'values': {
+            'hostPathVolumes': [
+                {
+                    'hostPath': f'/mnt/{pool_name}/tc-hostpath',
+                    'mountPath': f'/mnt/{pool_name}/tc-mountpath',
```

Comment:
We have already mounted at this path inside the pod ?
`mountPath` refers to the path inside the pod.